exonum-java-binding/core/rust/src/storage/temporarydb.rs

```diff
@@ -67,7 +68,8 @@ pub extern "system" fn Java_com_exonum_binding_core_storage_database_TemporaryDb
 ) -> Handle {
     let res = panic::catch_unwind(|| {
         let db = handle::cast_handle::<TemporaryDB>(db_handle);
-        Ok(handle::to_handle(View::from_owned_fork(db.fork())))
+        let access = ErasedAccess::from(unsafe { into_generic_raw_access(db.fork()) });
+        Ok(handle::to_handle(access))
```

Comment:
(here and elsewhere) Shall we extract this method in a similar to the previous one (ErasedAccess:from(fork))? We had quite nice methods, now we have more heavyweight code.